src/fixes.rs

```diff
@@ -102,6 +106,7 @@ pub fn get_fixes_without_resolving_overlapping(
                             end: TextOffset::START,
                         },
                         suggestion: imports_suggestion,
+                        description: String::new(),
```

Comment:
Would be good to add some kind of message like `"Add necessary import for {} import", fix_node.span(db)` or something like that